.github/workflows/license-check.yaml

```diff
@@ -0,0 +1,45 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
As part of #359, all http links were updated to https. However, this file still has http. I think Apache Skywalking requires further investigation.

Reply:
@ashvina Thank you for your reminder. It has been fixed, PTAL, thanks.

Reply:
This is still `http` but should be `https`?